src/main/scala/io/github/kiemlicz/shelm/ChartDownloader.scala

```diff
@@ -19,7 +19,25 @@ object ChartDownloader {
     * @param chartLocation Chart reference
     * @return directory containing Chart
     */
-  def download(chartLocation: ChartLocation, downloadDir: File, sbtLogger: Logger): File = {
+  def download(chartLocation: ChartLocation, downloadDir: File, cacheDir: Option[File], sbtLogger: Logger): File = {
+    def pullFromCacheOrRemote(chartName: ChartName, chartVersion: Option[String], downloadFunction: () => Unit) = cacheDir match {
+      case Some(cache) =>
+        pullFromCache(chartName, chartVersion, downloadDir, cache).map { chartDir => {
+          sbtLogger.info(s"Helm chart cache hit for ${chartName.name}")
+          chartDir
+        }
+        }.getOrElse({
```

Comment:
the `(` and maching `)` is unnecessary here

Reply:
done